cockatrice/src/dialogs/dlg_move_top_cards_until.cpp

```diff
@@ -0,0 +1,54 @@
+#include "dlg_move_top_cards_until.h"
+
+#include "trice_limits.h"
+
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QLineEdit>
+#include <QSpinBox>
+#include <QString>
+#include <QVBoxLayout>
+#include <QWidget>
+
+DlgMoveTopCardsUntil::DlgMoveTopCardsUntil(QWidget *parent, QString _expr, uint _numberOfHits) : QDialog(parent)
+{
+    exprLabel = new QLabel(tr("Card name (or search expressions):"));
+
+    exprEdit = new QLineEdit(this);
```

Comment:
now that we have this class for it that constructs the dialog, it would be nice to add the search syntax help we have on the deck editor somehow (`TabDeckEditor::showSearchSyntaxHelp`), this would mean moving that somewhere else as well so it might be a bit more work, doesn't have to be in this pr of cours

Reply:
yeah, something to keep in mind for the future